### ISSUE TITLE
catkin_make_isolated does not add plain cmake folders to CPP

### ISSUE THREAD

**Issue by tkruse** [ros/catkin#309]
This causes e.g. cv_bridge to fail to build. To reproduce, I'll use flann, as it's a bot faster to get and compile, and I am too lazy to create a dummy plain cmake proejct:

```
mkdir flann_ws
cd flann_ws/
mkdir src
echo "- git: {local-name: catkin, uri: 'https://github.com/ros/catkin.git', version: 'groovy-devel'}
- tar: {local-name: flann, uri: 'https://github.com/ros-gbp/flann-release/archive/release/flann/1.7.1.tar.gz',
    version: flann-release-release-flann-1.7.1}" >> src/.rosinstall
wstool update -j8 -t src
cd src/
catkin_create_pkg flanndep flann
rm flanndep/CMakeLists.txt
echo "cmake_minimum_required(VERSION 2.8.3)
project(foobar)
find_package(catkin REQUIRED)
find_package(flann REQUIRED)" > flanndep/CMakeLists.txt
cd ..
src/catkin/bin/catkin_make_isolated
```

yields:

```
CMake Error at CMakeLists.txt:4 (find_package):
  Could not find module Findflann.cmake or a configuration file for package flann.
```

consider CPP:

```
$ /tmp/flann_ws/devel_isolated/catkin/env.sh env | grep PREFIX
CMAKE_PREFIX_PATH=/tmp/flann_ws/devel_isolated/catkin:/home/kruset/local/ubuntu-amd64/
```

I think this misses flann_ws/devel_isolated/flann

**Comment by wjwwood** [comment]
This is valid, we didn't release a fixed version yet.

**Comment by wjwwood** [comment]
This should be resolved by #313, can you check and see if you can confirm this?  I have also committed a simple workspace to the fix_cmi_devel branch which shows this working (for C++ anyways).

**Comment by tkruse** [comment]
The steps above I used to reproduce now work, so I confirm. Close if you want.
